openfisca_us/variables/demographic/geographic/county_str.py

```diff
@@ -0,0 +1,12 @@
+from openfisca_us.model_api import *
+
+
+class county_str(Variable):
+    value_type = str
+    entity = Household
+    label = "County (string)"
+    documentation = "County variable, stored as a string"
+    definition_period = ETERNITY
+
+    def formula(household, period, parameters):
```

Comment:
Technically we don't actually need `formula` to be a function of `parameters` - could just be `def formula(household, period):`. But we're not consistent on this - @MaxGhenis do you think we should start doing this?

Reply:
Filed https://github.com/PolicyEngine/openfisca-us/issues/467 to decide and document. Let's leave as-is for now.